src/viser/_messages.py

```diff
@@ -409,7 +408,7 @@ class DirectionalLightProps:
     """Color of the directional light. Synchronized automatically when assigned."""
     intensity: float
     """Intensity of the directional light. Synchronized automatically when assigned."""
-
+    castShadow: bool
```

Comment:
can we do `cast_shadow` instead of `castShadow` to match the naming conventions we use on the Python API? a docstring would also be nice!